addon/components/fd-sheets/fd-navigation-sheet.js

```diff
@@ -57,6 +49,21 @@ export default FdBaseSheet.extend({
   */
   isAddMode: false,
 
+  /**
+    Custom buttons for `fd-sheets-tool-bar` on `fd-navigation-sheet` route.
+
+    @property customButtons
+    @type Array
+  */
+  customButtons: computed('i18n.locale', 'isAddMode', function() {
+    let i18n = this.get('i18n');
+    return [{
+      buttonVisible: this.get('isAddMode'),
+      buttonName: i18n.t('forms.fd-navigation.cancel'),
+      buttonAction: () => this.set('isAddMode', false),
```

Comment:
63:27  error  Don't introduce side-effects in computed properties  ember/no-side-effects